src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderUtils.ts

```diff
@@ -160,7 +160,7 @@ export const buildQuery = (querySnippets: QueryRowData[]) => {
     querySnippets.forEach((snippet, index) => {
         if (snippet.operatorData.operatorType === OperatorType.Function) {
             if (index !== 0) {
-                fullQuery = fullQuery.concat(snippet.combinator);
+                fullQuery = fullQuery.concat(snippet.combinator + ' ');
```

Comment:
a few lines later, we have string interpolation being used. Could we use that here for consistency?

Reply:
Good call, made the change